ui/install.slint

```diff
@@ -5,7 +5,7 @@
 import {
     ComboBox, VerticalBox, HorizontalBox, GridBox, Button,
     LineEdit, ListView, GroupBox, CheckBox
-} from "std-widgets.slint";
+, ProgressIndicator} from "std-widgets.slint";
```

Comment:
I guess the LSP created this terrible formatting?

Reply:
Heh yes. I'll make it less terrible :)